ios/Makefile

```diff
@@ -25,9 +34,41 @@ archive:
 		-scheme $(scheme_name) \
 		-archivePath $(archive_path)
 
-upload:
+export_ipa:
 	xcodebuild \
 		-exportArchive \
 		-archivePath $(archive_path) \
+		-exportPath ./build \
 		-exportOptionsPlist exportOptions.plist \
-		-allowProvisioningUpdates
+
+upload_ipa:
```

Comment:
Maybe add here internal dependencies? You can't run this unless `export_ipa` is run first, and that one depends on `archive`, right?

Reply:
You mean to call export_ipa inside upload_ipa?
The dependencies are expressed in the workflow file with steps. If we would want to have the full flow described in this Makefile, then we can have a single make command, like `upload_to_app_store` and execute everything at once.

Reply:
No, just use `make` internal handling of dependencies.

Reply:
Done, moved the dependencies to Makefile. I have removed the steps from the workflow.